### ISSUE TITLE
controller-as-route syntax rule should be improved

### ISSUE THREAD

**Issue by ank91** [Gillespie59/eslint-plugin-angular#336]
I have my angular routes in this way

```javascript

$routeProvider.when('/home', {
            templateUrl: 'views/home.html',
            controller: 'HomeController as homeCtrl'
        })
.otherwise({ redirectTo: '/' });
```

Above code works fine (angular v1.4.8) but does not pass ```controller-as-route``` [rule](https://github.com/Gillespie59/eslint-plugin-angular/blob/master/rules/controller-as-route.js#L32)
It should check if controller name has ```as``` in between.

**Comment by Gillespie59** [comment]
:+1:
